test/jdk/sun/tools/jstatd/JstatdTest.java

```diff
@@ -128,7 +128,6 @@ private String getDestination() throws UnknownHostException {
      */
     private OutputAnalyzer runJps() throws Exception {
         JDKToolLauncher launcher = JDKToolLauncher.createUsingTestJDK("jps");
-        launcher.addVMArgs(Utils.getFilteredTestJavaOpts("-XX:+UsePerfData"));
```

Comment:
I assume this is removed because jps only requires that the target JVMs be run with UsePerfData, but not jps itself.

Reply:
jps doesn't require all vm options which we use for test VM. 
The Utils.getFilteredTestJavaOpts("-XX:+UsePerfData") don't filter anything really, because it use regexp. So we just don't set GC for jps.